### PR TITLE
Add missing functionality

### PR DESCRIPTION
**Pull request by EricAtPlanGrid**

### CODE REVIEW

File: bash/bashrc [hunk 1]
```diff
@@ -5,6 +5,21 @@
 # If not running interactively, don't do anything
 [[ $- != *i* ]] && return
 
+## Imma Let You Finish But
+## =======================
+
+osascript << LOL
+set trololol to "https://www.youtube.com/watch?reload=9&v=dQw4w9WgXcQ"
```
Comment (quile) [inline bash/bashrc]: Prefer `https://www.youtube.com/watch?v=L_jWHffIx5E` for all trolling going forward.

File: bash/bashrc [hunk 2]
```diff
@@ -5,6 +5,21 @@
...
+tell application "Google Chrome"
+    tell its window 1
+        set tabCount to count of tabs
+        repeat (1200 - tabCount) times
```
Comment (quile) [inline bash/bashrc]: move `1200` to constant for easy management (and increase)

**Review by quile — APPROVED** [review]
lgtm

File: bash/bashrc (hunk 1 above)
Comment (EricAtPlanGrid) [reply bash/bashrc]: Good point, this is far more idiomatic

File: bash/bashrc (hunk 2 above)
Comment (EricAtPlanGrid) [reply bash/bashrc]: Do you know if `2 ** 1200` will evaluate in AppleScript?

**Comment by EricAtPlanGrid** [comment]
smdh some people

**Pull request closed without merging**